addons/portal/wizard/portal_share_views.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="portal_share_wizard" model="ir.ui.view">
+        <field name="name">portal.share.wizard</field>
```

Comment:
Here also: portal.share.view.form

I let you check remaining IDs if necessary :)